.travis.yml

```diff
@@ -6,14 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-## PHP versions we test against
-php:
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm
-
-## Build matrix for lowest and highest possible targets
```

Comment:
> Also you don't need to specify php versions when you put them in the build matrix

Actually those ensure lowest, current, and highest possible are tested, for reference: https://blog.wyrihaximus.net/2015/06/test-lowest-current-and-highest-possible-on-travis/